example_code/shmem_team_context.c

```diff
@@ -1,107 +1,80 @@
 #include <shmem.h>
 #include <stdio.h>
 
-int isum, ival;
 
-int my_ctx_translate_pe(shmem_ctx_t src_ctx, int src_pe, shmem_ctx_t dest_ctx) {
-  if (src_ctx == SHMEM_CTX_INVALID) {
-    return -1;
-  }
-  if (dest_ctx == SHMEM_CTX_INVALID) {
-    return -1;
-  }
-
-  shmem_team_t src_team, dest_team;
-  shmem_ctx_get_team(src_ctx, &src_team);
-  shmem_ctx_get_team(dest_ctx, &dest_team);
-  return shmem_team_translate_pe(src_team, src_pe, dest_pe);
-}
-
-shmem_ctx_t my_team_create_ctx(shmem_team_t team) {
-  if (team == SHMEM_TEAM_INVALID) {
-    return SHMEM_CTX_INVALID;
-  }
+int main(void)
```

Comment:
This whole example is bit convoluted for a simple team-based context and team-based PE numbering in an RMA operation. Not sure, if we can change this now:
1. Can we just create one team and use the team world instead of the ```team_3```
2. Am I missing some sort of a common usage model that the current example tries to provide?

Reply:
I'm not entirely sure we can make such changes either, but if so, I'm ok with removing `team_3`. Translating to the world team is probably a more common use case.  @jdinan, would this change be permissible?

Reply:
The existing teams examples sorely need attention and I fully support using section committee power to fix them. The hitch is that we need 100% consensus for any changes at this point to clear upcoming votes. Unfortunately, we're only going to get one shot at that, since these won't be read until the RC2 meeting. Here's what I propose -- let's do the best job that we can with these in advance of the RC1 meeting. @manjugv can we get some time at the end of that meeting to read these changes for feedback and ensure that they don't cause us to fail the RC2 vote?

Reply:
> @manjugv can we get some time at the end of that meeting to read these changes for feedback and ensure that they don't cause us to fail the RC2 vote?

👍